extension/src/cli/git/constants.ts

```diff
@@ -44,4 +47,4 @@ export enum Commit {
 
 export const DEFAULT_REMOTE = 'origin'
 
-export type Args = (Command | Flag | Commit | typeof DEFAULT_REMOTE)[]
+export type Args = (Command | Flag | Commit | typeof DEFAULT_REMOTE | string)[]
```

Comment:
I needed to be able to add unique `sha` ids as an argument but is there a better way to fix the type besides just allowing strings in general?